package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@hookform/resolvers": "^3.3.2",
+    "dotenv": "^16.3.1",
```

Comment:
**nitpick:** I think you don't need it if you use `VITE_APP` env

Reply:
For unknown to me reasons It's necessary for env to work .-.